keps/sig-api-machinery/2340-Consistent-reads-from-cache/kep.yaml

```diff
@@ -3,6 +3,7 @@ kep-number: 2340
 authors:
   - "@jpbetz"
   - "@wojtek-t"
+  - "@siarkowicz"
 owning-sig: sig-api-machinery
 participating-sigs:
```

Comment:
nit: can we add https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3157-watch-list to the related sigs since it helps prove the viability of the general approach

Reply:
Added to see-also.